packages/babel-helper-member-expression-to-functions/src/index.ts

```diff
@@ -562,6 +573,7 @@ export interface Handler<State> {
     member: Member,
     args: t.OptionalCallExpression["arguments"],
   ): t.Expression;
+  delete?(this: HandlerState<State> & State, member: Member): t.Expression;
```

Comment:
```suggestion
  // TODO(Babel 8): Consider making this required, since `.get` doesn't
  // really work as a fallback for `.delete`
  delete?(this: HandlerState<State> & State, member: Member): t.Expression;
```